mobile-app/app/components/NumberRow.tsx

```diff
@@ -95,6 +97,11 @@ export function NumberRow (props: INumberRowProps): JSX.Element {
             props.rhs.suffixType === 'component' &&
             (props.children)
           }
+          {
+            props.isOraclePrice === true && (
```

Comment:
Can't we just do `props.isOraclePrice`?

Reply:
eslint will complain @typescript-eslint/strict-boolean-expressions